### DISCUSSION TITLE
DietPi-Software | WebWormhole

### DISCUSSION THREAD

**Discussion by ingoiot** [MichaIng/DietPi#5769] [Software Request]
#### Formal software information
- Software title | WebWormhole 
- Short description | WebWormhole creates ephemeral pipes between computers to send files
or other data.

- Official URL (if available) | https://webwormhole.io/
- GitHub URL (if available) | https://github.com/saljam/webwormhole
- (Official) install documentation | https://github.com/saljam/webwormhole

#### Are there similar/alternative software titles available with DietPi-Software?
no
- ...

#### How can DietPi make the installation easier or compatible, than following the install instructions or do APT installation, if available?
 download and compile and start server on a free port.

**Comment by Ast3risk-ops** [comment]
Uhhhh, you mean wormhole, correct?

**Comment by Ast3risk-ops** [comment]
The file sharing thingy? This looks like a web interface for it

**Comment by Joulinar** [comment]
I'm a little bit worried on following statement done on their GitHub repository

> THIS PROJECT IS STILL IN EARLY DEVELOPMENT IT USES EXPERIMENTAL
CRYPTOGRAPHIC LIBRARIES AND IT HAS NOT HAD ANY KIND OF SECURITY
OR CRYPTOGRAPHY REVIEW THIS SOFTWARE MIGHT BE BROKEN AND UNSAFE

And nothing happened on the project for more than a year. 🤔

**Reply by ingoiot** [reply]
yes, i noticed that too, but if it works you don't have to update it.

i tried yesterday to compile file.pizza - a similar service under dietpi on a virtual box, unfortunately without success.
i suppose the enviroment(nodejs and npm) doesn't work with the old code anymore.

it wouldn't necessarily have to be webwormhole either,
but since these services are sometimes offline and for a little more security i would like to host the service myself.

**Reply by MichaIng** [reply]
> but if it works

If it's about security issues, "it works" is not enough. It works until someone hacks it and steals all your private data 😉. A year no development on an experimental version indeed doesn't look good. However, we may have a look onto how widely it is used despite this, and what others say about potential security risks.
